src/SigmaDSP.cpp

```diff
@@ -180,6 +180,24 @@ void SigmaDSP::volume(uint16_t startMemoryAddress, float dB)
   safeload_write(startMemoryAddress, volume);
 }
 
+/**
+ * @brief control mute/unmute block without slew
+ *
+ * @param startMemoryAddress DSP memory address
+ * @param flag true or false
+ */
+void SigmaDSP::mute(uint16_t startMemoryAddress, bool flag)
+{
+    if (flag)
```

Comment:
`8388608` looks like a "magic number". Please use 0x800000 instead, like I've done other places in the code.
Also, `flag` doesn't say much. Use `state` (as in mute state) instead.

And I personally think the code is a bit verbose:
This is what I would prefer:
```cpp
/**
  * @brief Control mute/unmute block without slew
  *
  * @param startMemoryAddress DSP memory address
  * @param state true or false
  */
void SigmaDSP::mute(uint16_t startMemoryAddress, bool flag)
{
  safeload_write(startMemoryAddress, state ? 0x800000 : 0);
}
```

Reply:
Cheers for the feedback on this, completely forgot about ternary expressions. I've resolved this comments now

Reply:
No worries! Even though long and nested ternary expressions are difficult to read, they're neat for small and obvious things like flags!